R/calcShareOdyssee.R

```diff
@@ -88,6 +93,17 @@ calcShareOdyssee <- function(subtype = c("enduse", "carrier", "enduse_carrier"),
     interpolate_missing_periods(expand.values = TRUE)
 
 
+  # Use old data where new demand is zero
+  odyssee <- odyssee %>%
+    select(-"model", -"scenario", -"unit") %>%
+    pivot_wider(names_from = "version", values_from = "value") %>%
+    group_by(across(-all_of(c("new", "old")))) %>%
+    mutate(value = ifelse(all(.data$new == 0), .data$old, .data$new)) %>%
+    ungroup() %>%
+    select(-"old", -"new") %>%
```

Comment:
Very minor hint to a need argument of mutate :)
```suggestion
    mutate(value = ifelse(all(.data$new == 0), .data$old, .data$new), .keep = "unused") %>%
    ungroup() %>%
```